app/assets/stylesheets/components/_map.scss

```diff
@@ -114,4 +114,15 @@
       }
     }
   }
+}
+
+#wrapper { 
+  position: relative;
+}
+
+#over_map { 
+  position: absolute; 
+  top: 35px; 
+  left: 35%; 
+  z-index: 99;
 }
```

Comment:
What's this for?

Reply:
<img width="1410" alt="Screen Shot 2022-01-08 at 22 21 19" src="https://user-images.githubusercontent.com/63218144/148645743-611c6ecb-d83c-4a7d-acb8-1d093f189232.png">

map and div on it